src/lib.rs

```diff
@@ -1057,8 +1083,21 @@ mod test {
     }
 
     pub fn rng(seed: u64) -> TestRng<StdRng> {
-        let seed = [seed as usize];
-        TestRng { inner: StdRng::from_seed(&seed) }
+        // TODO: use from_hashable
+        let mut state = seed;
+        let mut seed = <StdRng as SeedableRng>::Seed::default();
+        for x in seed.iter_mut() {
+            // PCG algorithm
```

Comment:
We don't really need PCG here, simply using the bytes is enough (for ISAAC of HC-128). But it is temporary anyway.